web/satellite/src/components/infoBars/PaidTierBar.vue

```diff
@@ -6,20 +6,30 @@
         <p>
             Upload up to 75TB by upgrading to a Storj Pro Account.
         </p>
-        <p class="pt-bar__functional" @click.stop="openAddPMModal">
+        <p class="pt-bar__functional" @click.stop="openAddPMModal" @click="openBanner">
```

Comment:
you could place `openAddPMModal` call inside `openBanner` function to have only one `@click` handler. That would make things more clean

Reply:
Done